layouts/index.html

```diff
@@ -53,7 +53,7 @@ <h2>{{ i18n "sectionTitle1" }}</h2>
     <div class="card">
       <h2>{{ i18n "sectionTitle2" }}</h2>
       <p>{{ i18n "sectionText2" | markdownify }}</p>
-      <a href="https://coreruleset.org/docs/concepts/" class="button"
+      <a href="https://github.com/owasp-modsecurity/ModSecurity/wiki" class="button"
```

Comment:
This modification is part of another PR - perhaps you forget to check out a new branch? (I don't know if this will conflict with another PR)

Reply:
Thanks - yes - I forgot to switch to the `main`. It has been fixed.